### PR TITLE
fix missing double quotes in scl_source

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: shell/scl_source [hunk 1]
```diff
@@ -66,7 +66,7 @@ if [ $_recursion == "false" ]; then
         else
             export X_SCLS="${_scls[$_i]} $X_SCLS"
         fi;
-        _i=$(($_i+1))
+        _i=$(("$_i"+1))
```
Comment (phracek) [inline shell/scl_source]: why double brackets?

Comment (praiskup) [reply shell/scl_source]: That's bash syntax for numeric expressions?

**Comment by phracek** [comment]
LGTM. Thanks

File: shell/scl_source (hunk 1 above)
Comment (praiskup) [reply shell/scl_source]: Ah, here should be just `_i=$(( _i + 1 ))`.

**Comment by phracek** [comment]
What about to write a test for it?

**Comment by phracek** [comment]
@praiskup Does it make sense to you to write a test?

**Comment by praiskup** [comment]
This was mostly per shellcheck;  so yes, it would make sense to add test for
shellcheck issues (as is done in container-common-scripts).  But honestly,
I won't have enough time on this topic.

**Comment by dmnks** [comment]
Pavel, I've rebased your patch on master and merged it. Thank you for your contribution!

**Pull request closed without merging**